### PR TITLE
added my e3 to sw mod

### PR DESCRIPTION
**Pull request by UltraWaffles**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### How was it tested? 
i made the printer and obtained a serial
#### Further notes
I added "supports" to the parts that needed one, not sure if that's allowed

### CODE REVIEW

**Comment by KiloQubit** [comment]
So we already have two mods for converting an ender 3 to a switchwire:

- https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Gizzle/ender-3_(pro)_switchwire
- https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Triano/Ender_3Pro_Switchwire

How is this one different or better than the two we already have?

**Comment by UltraWaffles** [comment]
> So we already have two mods for converting an ender 3 to a switchwire:
> 
> * https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Gizzle/ender-3_(pro)_switchwire
> * https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Triano/Ender_3Pro_Switchwire
> 
> How is this one different or better than the two we already have?

It allows for the original closure idea to still work. It also allows for the original ender 3 motors to work with the motor mounts. I made this mod a while ago ([here](https://github.com/UltraWaffles/Voron-Switchwire/tree/master/Mods) (I think before those 2 were completed), just didnt have time to submit it to user mods.

**Comment by KiloQubit** [comment]
The Triano mod also allows the motors to be reused.  Unfortunately I don't believe there's enough to differentiate your mod from the ones that we already have, so I'm going to close this PR.  Please reopen if you believe this to be in error.

**Comment by UltraWaffles** [comment]
> The Triano mod also allows the motors to be reused. Unfortunately I don't believe there's enough to differentiate your mod from the ones that we already have, so I'm going to close this PR. Please reopen if you believe this to be in error.

triano uses new motors, if his cad is correct. I guess enclosures arent important enough to be considered huh?

**Comment by UltraWaffles** [comment]
Not sure if you could tell,  but the extrusions are 2020s and 2040s only #

**Comment by KiloQubit** [comment]
After looking at your mod more, I think it can qualify as not a duplicate.  I have not reviewed this PR yet, but some things I see that need to be changed:

 - update `printer_mods/README.md` to note in the description that this mod allows the printer to be enclosed
 - remove spaces from all filenames
 - upload individual STLs, not a RAR (or any other archive) of them
 - if any of the original Voron parts are in the STLs, remove them and make a note in the README pointing users to the original parts in the Voron-Switchwire repository

**Comment by UltraWaffles** [comment]
i have done what u requested

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please remove spaces from all filenames and directory names (CAD, DXF, etc - all of them)

It would be nice to include a photo of the built printer, and not just a CAD render.

Rename the xlsx file to something that has BOM in its name, to make it clear what it is.

`rear_screen.stl` and `front_display.stl` have an overhang over 45 degrees on the bottom due a filet.  You can fix it by replacing it with a chamfer:

Screen Shot 2021-06-30 at 15 58 59

Remove `2020_MGN12_guide_x2.stl`, since that's a stock Voron part.

`z_motor_mount_a.stl` is not printable in the provided orientation

Screen Shot 2021-06-30 at 16 11 04

`y_belt_anchor.stl` - is that a built-in support?  If so, add a note to the README for users to remove it after printing.

**Comment by UltraWaffles** [comment]
I think i done all of the things you've asked, please check if theres anything i've missed

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!

There are still a bunch of files and directories that have spaces in their names, please remove the spaces.  E.g. `Electronics/screen_(ender3_original) wip`, `STLs/XZ_axis/Z_bearing block left.stl`, etc.

`screen_rear.stl` is not in a printable orientation

`front_display.stl` still has a filet on the bottom, which results in a too steep overhang.

`Z_bearring_block right.stl` has an extra `r` in the name

A few things in the README are still not fixed (STLs files to STL files, execpt, PSU, link to original STLs)

File: printer_mods/Ultrawaffles/Ender_3_to_SW/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+4. This design is made for 2020/2040 extrusions
+5. You will be using alot of parts from the original ender 3
+6. I have a BoM for all the parts you need
+7. Hammerhead tnuts works just as well as those expensive sliding ones.
+8. if you buy shorter M4 flathead screws, you can reduce the height of the bed, thus creating more z height.
+9. This version allows you to enclose it without any problems
+10. For now, the screen and door panel conflicts with each other, Ill fix it when I find a solution
+11. Not sure if im allowed to do this, but all of the files (execpt for electronic mounts) are in the stls.rar file
+
+STLs files:
```
[the remark this answers is not in the document]
Comment (KiloQubit) [reply printer_mods/Ultrawaffles/Ender_3_to_SW/README.md:20]: Still not fixed.

File: printer_mods/Ultrawaffles/Ender_3_to_SW/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+This mod uses some combination of both original stls and my own edited versions. The following list is parts that are unchanged. All electronic mounts execpt for the massive psu are from the v2.2
```
[the remark this answers is not in the document]
Comment (KiloQubit) [reply printer_mods/Ultrawaffles/Ender_3_to_SW/README.md:21]: Still not fixed

**Comment by UltraWaffles** [comment]
I updated the stls and typos, please take a peek

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!

Some of the README is still not fixed, please read my unresolved comment again.

The following files still have spaces in their names:

- `[a]_upper_idler_support_b left.stl`
- `[a]_upper_idler_support_b right.stl`
- `upper_idler_support_a LEFT.stl`
- `upper_idler_support_a RIGHT.stl`

**Comment by UltraWaffles** [comment]
I have updated readme, added link, and renamed stl files again. Hopefully I got it right this time

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**